docs/source/conf.py

```diff
@@ -371,13 +371,11 @@ def _auto_rst_for_module(module: types.ModuleType, exclude_members: List[Any]) -
 
 
 def _modules_to_rst() -> List[types.ModuleType]:
-    """Return the list of modules for which to generate API reference rst
-    files."""
-
+    """Return the list of modules for which to generate API reference rst files."""
     document_modules: List[types.Module] = [
         streaming,
     ]
-    exclude_modules: List[types.Module] = []
+    exclude_modules: List[types.Module] = [streaming.base, streaming._version]
```

Comment:
why? there is a lot of documented code under base which does not happen to be imported to top-level

Reply:
also the docs are missing for streaming.text, streaming.vision, we need to fix this too (__all__?)

Reply:
Added text and vision to a doc in https://github.com/mosaicml/streaming/pull/20/commits/b9e4a2d410dea27e5b08949e7ddca132ee29444d. Looks like adding all the class and method as part of `base` require some thorough investigation. I will do that in a separate PR.